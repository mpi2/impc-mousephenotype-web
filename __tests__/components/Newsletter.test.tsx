import { render } from "@testing-library/react";
import Newsletter from "@/components/Newsletter";

describe("Newsletter component", () => {
  it("renders correctly", () => {
    const { container } = render(<Newsletter />);
    expect(container).toMatchSnapshot();
  });
});
