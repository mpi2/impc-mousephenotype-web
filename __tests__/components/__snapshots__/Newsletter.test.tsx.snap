// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newsletter component renders correctly 1`] = `
<div>
  <div
    class="news-letter pt-5 pb-5 bg-orange"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-12 col-md-8 offset-md-2 text-center"
        >
          <h2>
            The IMPC Newsletter
          </h2>
          <p
            class="mt-4"
          >
            Get highlights of the most important data releases, news and events, delivered straight to your email inbox
          </p>
          <a
            class="btn btn-mailing btn-primary-dark text-white"
            href="https://forms.office.com/Pages/ResponsePage.aspx?id=jYTJ3EdDnkKo2NytnQUzMV3vNFn2DMZLqTjqljGsCfNUMDg0Q09OQThJUkVaVUpONVpSTVVSVEZERy4u"
            target="_blank"
          >
            Subscribe to newsletter
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
